torchrl/objectives/costs/ddpg.py

```diff
@@ -30,19 +30,24 @@ class DDPGLoss(_LossModule):
         device (str, int or torch.device, optional): a device where the losses will be computed, if it can't be found
             via the value operator.
         loss_function (str): loss function for the value discrepancy. Can be one of "l1", "l2" or "smooth_l1".
+        delay_actor (bool): whether to separate the target actor networks from the actor networks used for data
+            collection.
+        delay_value (bool): whether to separate the target value networks from the value networks used for data
```

Comment:
I know I did not do it for the other optional params, but perhaps you could add `(bool, optional)` and indicate the default value?

Reply:
Sure, these are now added in the latest commit.